rascontrol/rascontrol.py

```diff
@@ -406,38 +580,36 @@ def is_output_current(self, plan, show=False):
             print '>>>', (result, unknown_bool, message)
         return result, message
 
-    def _load_xs_list(self):
-        """ 
-        Returns list of all cross sections (Node objects)
-        """
-        xs_list = []
-        rivers = self.get_rivers()
-        for riv in rivers:
-            for reach in riv.reaches:
-                for node in reach.nodes:
-                    # blank node type indicates XS
-                    if node.node_type == '':
-                        xs_list.append(node)
-        return xs_list
-
     # Methods below here are semi-private and are intended to be called from the River, Reach, and Node classes
-    def output_getnodes(self, river_id, reach_id):
+    def geometry_getreaches(self, river_num):
         """
-        Return node names (stationing) and node types
-        Node types may belong to the following non inclusive list: '' (cross section), 'BR', 'Culv', 'IS', ...
-        :return: nodes_ids, node_types - two lists of strings
+        Returns reach names in river numbered river_num using Geometry_GetReaches
+        :param river_num: int
+        :return: list of reach names
         """
-        _, _, _, node_ids, node_types = self.com_rc.Geometry_GetNodes(river_id, reach_id, None, None, None)
-        return node_ids, node_types
-
+        
+        _, _, reaches = self.com_rc.Geometry_GetReaches(river_num, None, None)
+        return reaches
+    
     def output_getreaches(self, river_num):
         """
-        Returns reach names in river numbered river_num
+        Returns reach names in river numbered river_num using Output_GetReaches
         :param river_num: int
         :return: list of reach names
         """
+        
         _, _, reaches = self.com_rc.Output_GetReaches(river_num, None, None)
         return reaches
+    
+    def geometry_getnodes(self, river_id, reach_id):
+        """
+        Return node names (stationing) and node types
+        Node types may belong to the following non inclusive list: '' (cross section), 'BR', 'Culv', 'IS', ...
+        :return: nodes_ids, node_types - two lists of strings
+        """
+        _, _, _, node_ids, node_types = self.com_rc.Geometry_GetNodes(river_id, reach_id, None, None, None)
+
```

Comment:
I'm generally trying to keep blank lines to minimum and only use them were they help differentiate different methods, classes, or significant blocks of code. Please remove this line and others like it. Blank lines are allowed, but should be minimized.